emission/integrationTests/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "3"
+services:
+  web-server:
+    build: ../../integrationTests
+    depends_on:
+      - db
+    environment:
+      - DB_HOST=db
+      - WEB_SERVER_HOST=0.0.0.0
+    # ports:
+    #   - "8080:8080"
+    volumes:
+      # specify the host directory where the source code should live
+      # If this is ~/e-mission-server-docker, then you can edit the files at
+      # ~/e-mission-server-docker/src/e-mission-server/emission/...
+      # - CHANGEME:/src/
+      - ..:/src/e-mission-server
+    networks:
+       - emission
+  db:
+    image: mongo:4.4.0
+    deploy:
+      replicas: 1
+      restart_policy:
+        condition: on-failure
+
+    #Volumes is the preferred way to persist data generated by a container. In this case we use a volume to persist the contents
+    #of the data base. Learn more about how to use volumes here: https://docs.docker.com/storage/volumes/
+    # And learn how to configure volumes in your compose file here: https://docs.docker.com/compose/compose-file/#volume-configuration-reference
+    volumes:
+      - mongo-data:/data/db
+    networks:
+       - emission
+  #adding section to incorporate nominatim server functionality
+  nominatim:
+    image: mediagis/nominatim:4.2
+    container_name: norcal-nominatim
+    environment:
+      - PBF_URL=https://download.geofabrik.de/north-america/us/california/norcal-latest.osm.pbf
+      - REPLICATION_URL=https://download.geofabrik.de/north-america/us/california/norcal-updates/
+      - IMPORT_WIKIPEDIA=false
```

Comment:
why are we still using norcal? I thought you wanted to switch to Rhode Island since norcal was too large